apps/api/src/app/workflows-v2/e2e/workflow-test-data.e2e.ts

```diff
@@ -133,22 +133,15 @@ describe('Workflow Test Data', function () {
       workflowId: `test-workflow-${Date.now()}`,
       __source: WorkflowCreationSourceEnum.EDITOR,
       active: true,
-      steps: steps.map((step, index) => ({
+      steps: steps.map(({ type, ...rest }, index) => ({
+        ...rest,
         name: `Test Step ${index + 1}`,
-        type: step.type,
+        type,
       })),
     };
 
     const { body } = await session.testAgent.post('/v2/workflows').send(createWorkflowDto);
-    const workflow = body.data;
-
-    for (const [index, step] of steps.entries()) {
```

Comment:
This endpoint doesn't exist anymore. So the data generator of the E2E had to be updated